src/bandit-evaluator.spec.ts

```diff
@@ -0,0 +1,300 @@
+import { BanditEvaluator } from './bandit-evaluator';
+import {
+  BanditCategoricalAttributeCoefficients,
+  BanditModelData,
+  BanditNumericAttributeCoefficients,
+} from './interfaces';
+import { Attributes } from './types';
+
+describe('BanditEvaluator', () => {
+  const banditEvaluator = new BanditEvaluator();
+
+  // We don't want these methods part of the public interface, however it's handy to be able to test them individually
```

Comment:
Let me know if this is getting too fancy with TypeScript and I should just make them `public` 😅